de.dlr.sc.virsat.model.extension.requirements.ui/src/de/dlr/sc/virsat/model/extension/requirements/ui/snippet/UiSnippetSectionExistenceVerification.java

```diff
@@ -21,4 +31,24 @@
  * 
  */
 public class UiSnippetSectionExistenceVerification extends AUiSnippetSectionExistenceVerification implements IUiSnippet {
+	
+	@Override
+	public void createSwt(FormToolkit toolkit, EditingDomain editingDomain, Composite composite, EObject initModel) {
+		super.createSwt(toolkit, editingDomain, composite, initModel);
+		initializeHelperForModel(initModel);
+
+		Composite sectionBody = createSectionBody(toolkit, SECTION_HEADING, null, 1);
+		sectionBody.setLayout(new GridLayout(UI_LAYOUT_NR_COLUMNS, false));
+
+		Category viewerCategory = acHelper.getCategory(conceptId, categoryId);
+
+		// Don't show property elementsToBeVerified as it is not relvant for the user (only for verification)
+		for (AProperty property : ActiveConceptHelper.getNonArrayProperties(viewerCategory)) {
+			if (!property.getName().equals(ExistenceVerification.PROPERTY_ELEMENTTOBEVERIFIED)) {
+				createCommonPropertyWidgets(toolkit, sectionBody, property);
+				createCustomPropertyWidgets(toolkit, editingDomain, sectionBody, property);
+			}
+		}

```

Comment:
This fragment copied from `AUiSnippetGenericPropertyInstances` will be executed twice because of a `super` call.

Let's instead override `createCommonPropertyWidgets` and `createCustomPropertyWidgets` to check if we are in this particular property.